chapter3/exercise3.js

```diff
@@ -0,0 +1,16 @@
+(function() {
+
+    function countBs(sequenceCharacters) {
+        var counter = 0, result;
+        
+        for(var i = 0, lengthOfSequence = sequenceCharacters.length; i < lengthOfSequence; i++) {
+            if(sequenceCharacters[i] == "B"){
+                counter += 1;
+            }
+        }
+        result = counter;
```

Comment:
po co w ogole wprowadzasz tutaj zmienna result skoro caly czas operujesz na counter a na koniec tylko przypisujesz do result, zeby to linie pozniej zwrocic?

Reply:
Tak jak wcześniej napisałem, chciałem zrobić kod bardziej czytelny dla innych, nie bać używać się zmiennych i ich nazewnictwa. Typowy przyrost formy nad treścią. Poprawiłem :)